src/goto-programs/remove_function_pointers.cpp

```diff
@@ -131,9 +101,10 @@ remove_function_pointerst::remove_function_pointerst(
   }
 }
 
-bool remove_function_pointerst::arg_is_type_compatible(
+static bool arg_is_type_compatible(
```

Comment:
Is there a reason for preferring static functions over private member functions?

Reply:
As I made `remove_function_pointer` a global function, I had to have a way to use these functions.